dev/benchmark/sysbench.md

```diff
@@ -30,19 +30,19 @@ IDC 机器
 | OS       | linux (CentOS 7.3.1611)       |
 | CPU | 40 vCPUs, Intel(R) Xeon(R) CPU E5-2630 v4 @ 2.20GHz |
 | RAM | 128GB |
-| DISK | 1.5T SSD * 2  + Optane SSD * 1 |
+| DISK | 1.5T SSD \* 2  + Optane SSD \* 1 |
```

Comment:
Is "\" the escape character? Why we need it here?

Reply:
`*` sign is used to *bold* the text in markdown, so it is recommended to use `\` to escape the `*` sign

Reply:
see [MD037](https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md037---spaces-inside-emphasis-markers)